interpreter/cling/tools/packaging/cpt.py

```diff
@@ -518,8 +518,8 @@ def make(self, targets, flags=''):
                 exec_subprocess_call('%s --build . --target %s %s'
                                      % (CMAKE, target, flags), LLVM_OBJ_ROOT)
         else:
-            if args['verbose']: flags += ' VERBOSE=1'
-            exec_subprocess_call('make -j%d %s %s' % (self.cores, targets, flags),
+            if args['verbose']: exec_subprocess_call('cmake -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON', LLVM_OBJ_ROOT)
```

Comment:
Why we need `exec_subprocess_call` rather than just adding `-DCMAKE_VERBOSE_MAKEFILE=ON` to the `flags`.

Reply:
Then in that case we need to move this to `llvm_flags` variable...

Reply:
Alright I'll add that

Reply:
Hi @vgvassilev I have added the verbose flag to the llvm_flags variable in the download_llvm_binary function